modules/efmigration/lib/js/efmigration.jsx

```diff
@@ -122,6 +128,12 @@
                 currentStep.status = STEP_STATUS_RUNNING;
                 this.updateStep(currentStep);
 
+                // Hide migration intro text pp-migration-intro-text
```

Comment:
@olatechpro this works, but since it is a react component, we should not mix direct manipulation of DOM in it. I suggest you use a conditional rendering in the output of the component, or move the text into the same conditional that renders the migration warning. Something like this:

![image](https://github.com/publishpress/PublishPress-Planner/assets/407131/9fc1370d-3039-4096-936d-143908914b51)

Reply:
Thanks @andergmartins

This issue has been corrected now.